scripts/dependencies/check_transitive_dependencies.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+
+require_relative 'generate_dependencies'
+
+require 'fileutils'
+
+def execute_or_fail(command)
+    system(command) or raise "Failed to execute #{command}"
+end
+
+modules = [
```

Comment:
I think we should just do this for every module. But don't have overly strong feelings about it.